docs/docsite/rst/user_guide/playbooks_async.rst

```diff
@@ -35,8 +45,21 @@ poll value is 10 seconds if you do not specify a value for `poll`::
   task when run in check mode. See :doc:`playbooks_checkmode` on how to
   skip a task in check mode.
 
-Alternatively, if you do not need to wait on the task to complete, you may
-run the task asynchronously by specifying a poll value of 0::
+
+Concurrent tasks: poll = 0
+--------------------------
+
+When `poll` is 0, Ansible will start the task and immediately move on to the next one without waiting for a result.
+
+From the point of view of sequencing this is asynchronous programming: tasks may now run concurrently.
+
+The playbook run will end without checking back on async tasks.
+
+The async tasks will run until they either complete, fail or timeout according to their `async` value.
+
+If you need a synchronization point with a task, register it to obtain its job ID and use the `async_status` module to observe it.
```

Comment:
Link to module doc for async_status

Reply:
Thank you, I had thought of it but wasn't sure how to format the reference. Hope this is the right way.